036-kraft-mode.md

```diff
@@ -0,0 +1,103 @@
+# KRaft support: ZooKeeper-less Kafka
+
+## Motivation
+
+The Apache Kafka project is working on removing its dependency on ZooKeeper.
+Because of the complexity of this task, the work is in progress already for more than one year.
+It was stared with the [Kafka Improvement Proposal 500 (KIP-500)](https://cwiki.apache.org/confluence/display/KAFKA/KIP-500%3A+Replace+ZooKeeper+with+a+Self-Managed+Metadata+Quorum) and will continue for at least six to twelve months until all work is done and it is production-ready.
+Step by step, it removes the different dependencies on ZooKeeper.
+From the smaller things such as CLI tools or management APIs up to the complicated tasks such as quorum management and metadata management.
+It is based on _Apache Kafka Raft_ consensus protocol, which is also called _KRaft_.
+This is what gives the name to this proposal as well as to the feature gate discussed later.
+
+Strimzi aims to support the ZooKeeper-less Kafka and the KRaft protocol.
```

Comment:
Might be useful to add this issue link for people who want to be able to check on the overall status of the KRaft support: https://github.com/strimzi/strimzi-kafka-operator/issues/5615